Ginger/GingerCoreTest/AutoPilot/APILearningTests.cs

```diff
@@ -65,6 +66,29 @@ public void APILearnWSDLStockSimbolTest()
             Assert.AreEqual(AAMSList[0].AppModelParameters[1].PlaceHolder, "{LICENSEKEY}", "Is parameter name equal");
         }
 
+        [TestMethod]  [Timeout(60000)]
+        public void APIComparisonWSDLStockSimbolTest()
```

Comment:
this is only the start or all unit test you planning (i hope not..)
please add unit tests on new class called "APIModelsCompareTest"